dd-sdk-android/src/main/kotlin/com/datadog/android/rum/gestures/DatadogGesturesTracker.kt

```diff
@@ -0,0 +1,55 @@
+package com.datadog.android.rum.gestures
+
+import android.app.Activity
+import androidx.core.view.GestureDetectorCompat
+import com.datadog.android.tracing.Tracer
+import java.lang.ref.WeakReference
+
+internal class DatadogGesturesTracker(val rumTracer: Tracer) :
```

Comment:
SHouldnt call it `DatadogClickTracker` as it does not track any other gestures ?

Reply:
We don't actually know. For now it is only Tap but product will bring more I am almost sure. I would rather give it a general meaning.